src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.hpp

```diff
@@ -305,6 +305,10 @@ void populateLoweringONNXClipOpPattern(
     mlir::RewritePatternSet &, mlir::TypeConverter &, mlir::MLIRContext *);
 void populateLoweringONNXCumSumOpPattern(
     mlir::RewritePatternSet &, mlir::TypeConverter &, mlir::MLIRContext *);
+void populateLoweringONNXDFTOpPattern(
+    mlir::RewritePatternSet &, mlir::TypeConverter &, mlir::MLIRContext *);
+void populateLoweringONNXDFTV17OpPattern(
+    mlir::RewritePatternSet &, mlir::TypeConverter &, mlir::MLIRContext *);
```

Comment:
Again, please remove all stuffs about DFTV17Op after it is decomposed into DFTOp.

Reply:
I simply removed the lines of code you mentioned DFTV17 should be removed from.... I will do a search to see if there is anything else. This is not added code, this was already there from the initial review you looked at and you never commented on this file just an FYI.

Reply:
Removed and I did a search... don't see anymore.